editor/editor_inspector.cpp

```diff
@@ -1226,6 +1264,17 @@ void EditorInspectorSection::fold() {
 #endif
 }
 
+void EditorInspectorSection::disable() {
```

Comment:
I'm not sure disabling a whole section is a common use case, I would keep this a per-property function.

Reply:
Understood.